src/test/java/org/openrewrite/openapi/swagger/SwaggerToOpenAPITest.java

```diff
@@ -130,4 +130,44 @@ class Example {
           )
         );
     }
+
+    @Test
+    void migrateSwaggerDefinitionsToOpenAPIDefinition() {
+        rewriteRun(
+          recipeSpec -> recipeSpec.afterTypeValidationOptions(TypeValidation.none()),
+          //language=java
+          java(
+            """
+              import io.swagger.annotations.Info;
+              import io.swagger.annotations.SwaggerDefinition;
+              import jakarta.ws.rs.core.MediaType;
+
+              @SwaggerDefinition(
+                basePath = "/api",
+                host="example.com",
+                info = @Info(title = "Example", version = "V1.0"),
+                consumes = { MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML },
+                produces = { MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML },
+                schemes = { SwaggerDefinition.Scheme.HTTP, SwaggerDefinition.Scheme.HTTPS })
+              class Example {
+              }
+              """,
+            """
+              import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+              import io.swagger.v3.oas.annotations.info.Info;
+              import io.swagger.v3.oas.annotations.servers.Server;
+              import jakarta.ws.rs.core.MediaType;
```

Comment:
Any idea to remove this? Using `maybeRemoveImport` is not proper here since the origin parameter type is actually `String[]`

Reply:
Hmm; have you considered or tried using `doAfterVisit(new RemoveUnusedImports().getVisitor())` ?

Reply:
I tried, it doesn't work in this case

Reply:
Hmm; usually when that fails there's dangling type references; might there be any references left over somehow?

Reply:
Had a closer look; turns out there were missing types related to the server annotation, which were masked by the `TypeValidation.none` added to the test. I removed that call, fixed the type information by setting the classpath correctly, and then added the RemoveUnusedImports as shown above, which only works when there are no missing types.